examples/lighting-app/bouffalolab/bl602/go

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+MATTER_BL_ROOT=$(pwd)
```

Comment:
It looks like this can only run from the `current directory` which is probably not ideal. You can get the script directory instead of using PWD.

However general feedback: please do not use custom scripts, integrate with build_examples and that will enforce nicely formatted things.